### DISCUSSION TITLE
Is there any way to see which DB is being used by an instance of Keycloak

### DISCUSSION THREAD

**Discussion by dheeraj-326** [keycloak/keycloak#13756] [Q&A]
I want to know whether my docker instance of Keycloak is using the right DB. I have set the environment variables correctly but I want to check that in Keycloak. Is there any means to do that through logs/UI/docker terminal etc?

**Comment by Ali-Javanmardi** [comment]
I think this can be even a security thread.
If by any unforeseen situation there is an untrusted db in background and someone else had access to that db, entire security can be vulnerable.
So I think there is a good reason to see database connection info in keycloak admin panel.

**Comment by dheeraj-326** [comment]
This question was asked while I was trying to figure out an issue with dockerized Keycloak 19.0.0. I have more or less confirmed a bug where the environment variables are being ignored and H2 being used regardless of what is set. I am planning to open another issue for the bug.
